source/customization.rst

```diff
@@ -174,30 +178,34 @@ Start by creating the file
 
    # /etc/ood/config/apps/dashboard/initializers/ood.rb
 
-   OodFilesApp.candidate_favorite_paths.tap do |paths|
-     # add project space directories
-     projects = User.new.groups.map(&:name).grep(/^P./)
-     paths.concat projects.map { |p| Pathname.new("/fs/project/#{p}")  }
+  OodFilesApp.candidate_favorite_paths.tap do |paths|
+    # add project space directories
+    projects = User.new.groups.map(&:name).grep(/^P./)
+    paths.concat projects.map { |p| FavoritePath.new(Pathname.new("/fs/project/#{p}"))  }
 
-     # add scratch space directories
-     paths << Pathname.new("/fs/scratch/#{User.new.name}")
-     paths.concat projects.map { |p| Pathname.new("/fs/scratch/#{p}")  }
-   end
+    # add scratch space directories
+    paths << FavoritePath.new(Pathname.new("/fs/scratch/#{User.new.name}"))
 
-- The variable ``paths`` is an array of ``Pathname`` objects that define a list
+    # Project scratch has an optional title field.
+    paths.concat projects.map { |p| FavoritePath.new(Pathname.new("/fs/scratch/#{p}"), title: "Scratch")  }
+  end
```

Comment:
The indenting got modified by accident here. The code blocks should be indented with the comment, lining up with the "c" of  the "code-block" word but they were shifted back 1 character

Reply:
Should it be though?  It seems more natural to just keep the 2 space (tabs) instead of offsetting the initial indent by 3, then using indents of 2 within the block.

Reply:
Yes. The outer indentation is following Sphinx/restructured conventions. The 2 spaces in the block is following Ruby conventions.